From: Lorenzo Bianconi <lorenzo@kernel.org>
Date: Mon, 11 Apr 2022 12:13:25 +0200
Subject: [PATCH] net: ethernet: mtk_eth_soc: use standard property for
 cci-control-port

Rely on standard cci-control-port property to identify CCI port
reference.
Update mt7622 dts binding.

Signed-off-by: Lorenzo Bianconi <lorenzo@kernel.org>
Signed-off-by: David S. Miller <davem@davemloft.net>
---

--- a/arch/arm64/boot/dts/mediatek/mt7622.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt7622.dtsi
@@ -962,7 +962,7 @@
 		power-domains = <&scpsys MT7622_POWER_DOMAIN_ETHSYS>;
 		mediatek,ethsys = <&ethsys>;
 		mediatek,sgmiisys = <&sgmiisys>;
-		mediatek,cci-control = <&cci_control2>;
+		cci-control-port = <&cci_control2>;
 		mediatek,wed = <&wed0>, <&wed1>;
 		mediatek,pcie-mirror = <&pcie_mirror>;
 		mediatek,hifsys = <&hifsys>;
--- a/drivers/net/ethernet/mediatek/mtk_eth_soc.c
+++ b/drivers/net/ethernet/mediatek/mtk_eth_soc.c
@@ -3187,7 +3187,7 @@ static int mtk_probe(struct platform_dev
 		struct regmap *cci;
 
 		cci = syscon_regmap_lookup_by_phandle(pdev->dev.of_node,
-						      "mediatek,cci-control");
+						      "cci-control-port");
 		/* enable CPU/bus coherency */
 		if (!IS_ERR(cci))
 			regmap_write(cci, 0, 3);
